dtests/src/test/java/io/snappydata/hydra/consistency/consistency_putIntoUsingSparkApp.conf

```diff
@@ -0,0 +1,4 @@
+INCLUDE $JTESTS/io/snappydata/hydra/consistency/consistencyUsingApp.inc;
+
+io.snappydata.hydra.testDMLOps.SnappySchemaPrms-dmlOperations = ONEOF putinto FOENO;
```

Comment:
Currently, putInto doesn't have keys that will be used for update. We must have that scenarios covered otherwise we will be missing some code path.